src/main/java/org/jabref/model/database/BibDatabaseContext.java

```diff
@@ -249,7 +250,10 @@ public boolean hasEmptyEntries() {
     }
 
     public static Path getFulltextIndexBasePath() {
-        return Path.of(AppDirsFactory.getInstance().getUserDataDir(OS.APP_DIR_APP_NAME, SearchFieldConstants.VERSION, OS.APP_DIR_APP_AUTHOR));
+        return Path.of(AppDirsFactory.getInstance()
+                                     .getUserDataDir(OS.APP_DIR_APP_NAME,
+                                             "lucene" + File.separator + SearchFieldConstants.VERSION,
```

Comment:
Use     private static final String FILE_SEPARATOR = System.getProperty("file.separator");
 from OS.java in

Reply:
![image](https://github.com/JabRef/jabref/assets/1366654/c2ef868c-6b24-4202-94b2-2165bd6c5b09)

Reply:
I removed the unused variable. I didn't want to be the first one using it. 😇

Reply:
Then use the nio variant:         String separator = FileSystems.getDefault().getSeparator();